# モータフィードバック用カスタムメッセージ
string joint_name     # 関節名
float64 angle         # 角度 [rad]
float32 current       # 電流 [A]
int8 temperature      # 温度 